### DISCUSSION TITLE
Fileupload with custom action and view uploaded file in dirve

### DISCUSSION THREAD

**Discussion by dhruvil-chauhan-aavatto** [filamentphp/filament#12455] [Help] [answered]
### Package

Form builder

### Package Version

v3.2.57

### How can we help you?

I want to upload files in Google Drive and then get files from Google Drive and display them in an iframe. Is this possible or not?

**Comment by leandrocfe** [comment, marked as answer]
Maybe you can configure this package and use `disk('google')` in the FileUpload component 

https://github.com/yaza-putu/laravel-google-drive-storage

**Reply by dhruvil-chauhan-aavatto** [reply]
Thanks!
